sample/shared/shared/src/commonMain/kotlin/com/arkivanov/sample/shared/multipane/PreviewMultiPaneComponent.kt

```
@@ -0,0 +1,33 @@
+
+    @OptIn(ExperimentalDecomposeApi::class)
+    override val panels: Value<ChildPanels<Any, ArticleListComponent, Any, ArticleDetailsComponent, Any, Nothing>> =
+        MutableValue(
+            ChildPanels(
+                main = Child.Created(configuration = Unit, instance = PreviewArticleListComponent()),
+                details = if (isMultiPane) Child.Created(configuration = Unit, instance = PreviewArticleDetailsComponent()) else null,
+                mode = if (isMultiPane) ChildPanelsMode.DUAL else ChildPanelsMode.SINGLE,
+            )
+        )
+
+    override fun setMode(mode: ChildPanelsMode) {}
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)